nixpkgs.nix

```diff
@@ -7,7 +7,7 @@
 # The SHA256 will be printed as the last line of stdout.
 
 import ./fetch-nixpkgs.nix {
-    rev          = "fa12335f425808f53121713f501f3335878e6901";
-    sha256       = "1fjyvjxvymz8yd65ahgm798jp9vdcfy7s58zb5ns2iq2ak0h9j8p";
-    outputSha256 = "1qkihrm8xfrh93c7wh1d1x01p7mgv82b2ycpmn9jm5l7976g31vr";
+     rev          = "dd9f73e7d34486b09b966738ace161e621a0480b";
```

Comment:
While you are upgrading the Nixpkgs pin, could you switch to using `builtins.fetchTarball` as outlined here:

https://nixos.wiki/wiki/How_to_fetch_Nixpkgs_with_an_empty_NIX_PATH

… which will remove the need for `./fetch-nixpkgs.nix` and also remove the need for specifying two separate hashes.

Reply:
I'm not familiar with nix as a tool, so I will need some help sorting out this bit.
Trying something like:
```
let 
 nixpkgs = builtins.fetchTarball {
   url    = "https://github.com/NixOS/nixpkgs/archive/dd9f73e7d34486b09b966738ace161e621a0480b.tar.gz";
   sha256 = "0s674386v5b24a9fia26439gw9wsyhif85k2nzpxkp61293v3n3h";
 };

 in import nixpkgs {}
```
gives me issues elsewhere in `release.nix`

```
$ nix-shell --show-trace
error: while evaluating the attribute 'grpc-haskell.env' at /Users/tim/projects/gRPC-haskell/release.nix:221:5:
cannot coerce a set to a string, at /Users/tim/projects/gRPC-haskell/release.nix:211:16
```

Reply:
Fixed this issue.